carbon-design-system/carbon
# fix: respect max and min in NumberInput when using arrows

The arrow keys in the NumberInput allow users to step below the min and above the max value. This issue is fixed here.

#### Changelog

**Changed**

- Change the NumberInput to respect min and max values when using arrow keys

#### Testing / Reviewing

In order to reproduce the issue do the following:

- Create a NumberInput with min 0, max 100 and step 10

- manually enter the value 5 into the NumberInput

- click the arrow down.

actual behavior: the value of the NumberInput changes to -5

expected behavior: the value of the NumberInput changes to 0
